### PR TITLE
- Updated version 2.0.3

### PR DESCRIPTION
**Pull request by nk3nny**
- Tweaks to suppress to reduce suppression events slightly
- Fixed players not sorted in various leader manoeuvres
- Fixed more usage of gesture function
- Fixed prevents players from being ordered into guns
- Fixed prevents AI already manning vehicles from being ordered into weapons
- Tweaked fleeing state -- units prefer running standing
- Tweaked added more time for Suppression in FSM (prevents spam)
- Units tweaked for aggression in FSM movement (from 26,8 to 30,6)
- Tweaked manoeuvre cycles reduced from 4 to 2
- Tweaked manoeuvre frequency changed from 100 seconds to 70 seconds

-nb: suppression debug messages

### CODE REVIEW

**Comment by nk3nny** [comment]
nb, extra debug messages. wip.

File: addons/danger/functions/fnc_leaderManoeuvre.sqf [hunk 1]
```diff
@@ -42,14 +42,35 @@ _pos pushBack (_target call CBA_fnc_getPos);
...
         _x forceSpeed 2;
-        _x doMove selectRandom _pos;
+        _x doMove _pos;
 
         // force movement
-        if !(_x call FUNC(indoor)) then {_x playActionNow selectRandom ["FastF", "FastF", "FastLF", "FastRF"];};
+        if !(_x call FUNC(indoor)) then {[_x, ["FastF", "FastF", "FastLF", "FastRF"]] call FUNC(gesture);};
+        _x setVariable [QGVAR(currentTask), "Manoeuvre CQB"];
+
+        systemchat "MANOEUVRE CQC!";
```
Comment (jokoho48) [inline addons/danger/functions/fnc_leaderManoeuvre.sqf]: debug message missed

Comment (nk3nny) [reply addons/danger/functions/fnc_leaderManoeuvre.sqf]: Good spot. 

Just to be clear. I intend to remove all systemchat messages not wrapped in the debug variables before release. They are only here to excessively add feedback while testing this particular set of features.

File: addons/danger/functions/fnc_leaderManoeuvre.sqf [hunk 2]
```diff
@@ -17,71 +17,70 @@
...
 // find units
 if (_units isEqualTo []) then {
-    _units = units _unit;
+    _units = (units _unit) select {!isPlayer _x};
 };
 
-_unit setVariable [QGVAR(currentTarget), _target];
-_unit setVariable [QGVAR(currentTask), "Leader Manoeuvre"];
+// find overwatch position
+private _overwatch = [getpos _unit, ((_unit distance _target) / 2) min 300, 100, 8, _target] call lambs_danger_fnc_findOverwatch;
```
Comment (jokoho48) [inline addons/danger/functions/fnc_leaderManoeuvre.sqf]:
```suggestion
private _overwatch = [getpos _unit, ((_unit distance _target) / 2) min 300, 100, 8, _target] call FUNC(findOverwatch);
```

File: addons/danger/functions/fnc_leaderManoeuvre.sqf [hunk 3]
```diff
@@ -17,71 +17,70 @@
...
+    _units = _units select {alive _x && {!isPlayer _x}};
     _cycle = _cycle - 1;
 
     {
         // Half suppress -- Half manoeuvre
-        if (RND(0.6)) then {
-            _x forceSpeed 0;
+        if (RND(0.2) && {count _pos > 0}) then {
+            _x doWatch (_pos select 0);
+            [_x, _pos select 0] call lambs_danger_fnc_suppress;
```
Comment (jokoho48) [inline addons/danger/functions/fnc_leaderManoeuvre.sqf]:
```suggestion
            [_x, _pos select 0] call FUNC(suppress);
```

File: addons/danger/functions/fnc_leaderManoeuvre.sqf [hunk 4]
```diff
@@ -17,71 +17,82 @@
...
-        _x forceSpeed 2;
-        _x doMove selectRandom _pos;
-
-        // force movement
-        if !(_x call FUNC(indoor)) then {_x playActionNow selectRandom ["FastF", "FastF", "FastLF", "FastRF"];};
-
-    } foreach _units;
-};
+(group _unit) setFormDir (_unit getDir _target);
+if (_overwatch isEqualto []) then {(group _unit) move _target} else {(group _unit) move _overwatch;};
```
Comment (jokoho48) [inline addons/danger/functions/fnc_leaderManoeuvre.sqf]: this removes duplicated code
```suggestion
(group _unit) move ([_target, _overwatch] select (_overwatch isEqualto []);
```

**Pull request merged**